tools/swift-reflection-fuzzer/swift-reflection-fuzzer.cpp

```diff
@@ -131,6 +141,6 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
   auto reader = std::make_shared<ObjectMemoryReader>();
   NativeReflectionContext context(std::move(reader));
   context.addImage(RemoteAddress(Data));
-  context.getBuilder().dumpAllSections(std::cout);
+  context.getBuilder().dumpAllSections(stdout);
```

Comment:
Did the ABI change here and we just never built it? Why are you making this change here?

Reply:
Apparently it did. `dumpAllSections()` currently takes a `FILE *` and presumably at some point in the past it must have taken an `ostream&` or similar; either that, or the reflection fuzzer never worked, which I find hard to believe.

Reply:
It was changed in dd7d173c97ea "[stdlib] [reflection] Excise upstream from reflection".